api/PerformanceNavigation.json

```diff
@@ -118,10 +118,10 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": "7"
+              "version_added": false
```

Comment:
Based on the bug report (and playing with the API myself), I'm not sure that `redirectCount` matters at all. I'd undo this change